src/main/java/com/hms_networks/americas/sc/extensions/ExtensionsMain.java

```diff
@@ -0,0 +1,31 @@
+package com.hms_networks.americas.sc.extensions;
```

Comment:
Whats the reason for including a class with a main method?

Reply:
This main class/main method have no functionality. The Jars built by Maven include additional metadata like project version and name, but also the ability to double-click to launch the .jar using its main method. For consistency, and to not specify a main class which does not exist, I created this basic one.